src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -23,9 +26,20 @@ public void actionPerformed(ActionEvent e) {
 			case "Figure Joe 1":
 				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 				break;
+
 			case "Figure Joe 2":
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
+
+			case "Triangle":
+				FactoryInterface triangle = new TriangleFactory();
+				triangle.create().execute(driverManager.getCurrentDriver());
+				break;
+
+			case "Circle":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.